mypy_tests/check_negative_tests.py

```diff
@@ -0,0 +1,72 @@
+import os
+import re
+import subprocess
+from typing import Set
+
+OUTPUT_FILE = "negative.output"
```

Comment:
Why are you writing the output to a file instead of reading it from the standard output of the subprocess?

Reply:
Great point.  I was inspired by https://github.com/dropbox/mypy-protobuf, which uses a shell script for this.  Since we're in python there's no reason to not just capture the output directly.